### PR TITLE
get_detections_entities:

### PR DESCRIPTION
**Pull request by bdanjoux**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: CASE 03318743

## Description

The July commit https://github.com/demisto/content/commit/f53aa8711a013ebe4a893358dfad8f4519105ec7 related to then upcoming Crowdstrike Raptor release incorrectly updated the mechanism by which detection summaries are fetched.

The "legacy" API for fetching detection summaries: /detects/entities/summaries/GET/v1
was incorrectly amended to use /alerts/entities/alerts/v2 for clients who didn't tick the "legacy API" integration instance configuration boolean.

This was incorrect because the endpoint /detects/entities/summaries/GET/v1 is not deprecated in the CrowdStrike API documentation and also because /alerts/entities/alerts/v2 is for fetching alerts summaries, not detection summaries.

Subsequently, passing a valid detection id to an instance configured to use the new API endpoints fails when it works if you tick "use legacy API".

I removed mentions of LEGACY_VERSION and the incorrect new API endpoint.
I also corrected the argument description for cs-falcon-list-incident-summaries: ids
this command fetches incident details and subsequently takes incident IDs, not detection IDs

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py [hunk 1]
```diff
@@ -1590,9 +1590,9 @@ def get_detections_entities(detections_ids: list):
         :param detections_ids: IDs of the requested detections.
         :return: Response json of the get detection entities endpoint (detection objects)
     """
-    ids_json = {'ids': detections_ids} if LEGACY_VERSION else {"composite_ids": detections_ids}
-    url = '/detects/entities/summaries/GET/v1' if LEGACY_VERSION else '/alerts/entities/alerts/v2'
-    demisto.debug(f"Getting detections entities from {url} with {ids_json=}. {LEGACY_VERSION=}")
+    ids_json = {'ids': detections_ids}
+    url = '/detects/entities/summaries/GET/v1'
+    demisto.debug(f"Getting detections entities from {url} with {ids_json=}.")
```
Comment (amshamah419) [inline Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py:1595]: Hi @bdanjoux I'm not sure if we should remove the support for legacy version. Is there a specific issue you are experiencing?

Comment (bdanjoux) [reply Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py:1595]: Hi @amshamah419 I am experiencing an issue where an integration instance is not able to fetch detection summaries, that instance is configured with "use the legacy API" unticked.

As I explain in the PR description, the API endpoint /detects/entities/summaries/GET/v1 is not deprecated, it is very much current, please check the Crowdstrike API documentation.

This problem was introduced in the commit https://github.com/demisto/content/commit/f53aa8711a013ebe4a893358dfad8f4519105ec7    back in July.

in that July PR, many commands were updated as their endpoint was being deprecated and replaced with a new one, hence the LEGACY switch.

However, the command to fetch detection summaries should not have been modified, as its underlying endpoint was not deprecated. Additionally, what that still valid endpoint was replaced with, is an endpoint for alert summaries, not detection summaries.

This is why my integration instance is failing to retrieve detection summaries.

I input a valid detection ID to the detection summary fetching command, but the integration command feeds it to an endpoint that expects an alert ID instead.

In summary, I am not removing support for the legacy API endpoint, I am only trimming an incorrectly amended integration command.

Comment (amshamah419) [reply Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py:1595]: Hi @bdanjoux Can you please open an issue with CS? I think this is something which should be investigated since it could have effects elsewhere.

**Comment by amshamah419** [comment]
Hi @bdanjoux ,
Thank you again for opening a pull request. Unfortunately, we cannot accept it, as the change which was made originally was by design and works in our testing. If you are experiencing an issue with the command, please open a bug report with your CS so we can investigate further. 

Contributions are welcome, and we're looking forward to seeing you commit here again. Please let me know if you have any further questions.
